Havit/Havit/Network/Mock/MockParser.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MockParser.swift
+//  Havit
+//
+//  Created by SHIN YOON AH on 2022/01/14.
+//
+
+import Foundation
+
+final class MockParser {
+    static func load<T: Decodable>(type: T.Type, fileName: String) -> T? {
```

Comment:
p4;
사용할때 매번 `GenericResponse` 로 감싸기 귀찮으니까 아래처럼 선언부를 바꾸고
```suggestion
static func load<T: Decodable>(type: T.Type, fileName: String) -> GenericResponse<T>?
```
return 을 아래처럼 변경하면 
```swift
return try? JSONDecoder().decode(GenericResponse<T>.self, from: decodable)
```
사용하는 쪽에서 
```swift
let loadData = MockParser.load(type: [Content].self, fileName: "entire_content")
```
요렇게 사용할 수 있지 않을까여

Reply:
너무 좋은 생각입니다..🌝 반영해보겠습니다.!

Reply:
@sujinnaljin 
764a509 반영했습니다